polaris-patterns/src/components/AppProvider/AppProvider.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import type {AppProviderProps as PolarisAppProviderProps} from '@shopify/polaris';
+import {AppProvider as PolarisAppProvider} from '@shopify/polaris';
+
+export function AppProvider({children, ...rest}: PolarisAppProviderProps) {
+  return <PolarisAppProvider {...rest}>{children}</PolarisAppProvider>;
```

Comment:
I should probably decouple this like [online-store-ui](https://github.com/Shopify/online-store-ui#usage) and they make the consumer responsible for wrapping the ui libarary with the polaris provider. This means the library wouldn't work out of the box though without installing the core polaris library explicitly, which I feel weird about.